README.md

```diff
@@ -36,7 +36,7 @@ See [Testing Farm onboarding guide](https://docs.testing-farm.io/Testing%20Farm/
 | Input Name | Description | Default value |
 |------------|-------------|---------------|
 | `git_url` | An url to the repository with tmt metadata | url of repo that runs GHA |
-| `git_ref` | A tmt tests branch which will be used for tests | master |
+| `git_ref` | A tmt tests branch which will be used for tests | Pull Request base branch when run on `pull_request` or `pull_request_target` otherwise `master` |
```

Comment:
Good description.

Reply:
It should also work for other targets, such as issue_comment, but GitHub does not document it (as usual).